spec/support/login_helper.rb

```diff
@@ -4,4 +4,10 @@ def client_login
     login_as client, scope: :client
     client
   end
+
+  def employee_login
+    employee = create(:employee)
+    login_as employee, scope: :employee
+    employee
+  end
```

Comment:
Acho que não tinha rolado antes com o cliente mas aqui nesses métodos seria legal eles serem mais customizáveis algo como:

```suggestion
  def employee_login(employee = create(:employee))
    login_as employee, scope: :employee
    employee
  end
```

Porque aí se alguém precisar customizar algum campo no employee é só passar o argumento mas se não passar nada mantem o comportamento padrão, o que acha?

Reply:
Fica mais legal. Modificamos os métodos e implementamos nos outros testes.